components/brave_wallet/browser/account_discovery_manager.cc

```diff
@@ -163,4 +200,51 @@ void AccountDiscoveryManager::ProcessDiscoveryResult(
   }
 }
 
+void AccountDiscoveryManager::OnBitcoinDiscoverAccountsDone(
+    base::expected<DiscoveredBitcoinAccount, std::string> dicovered_account) {
+  if (!dicovered_account.has_value()) {
+    return;
+  }
+
+  auto& acc = dicovered_account.value();
+  if (acc.next_unused_receive_index == 0 && acc.next_unused_change_index == 0) {
+    return;
+  }
```

Comment:
What does this early return check mean, a comment would be helpful.

Reply:
added comment https://github.com/brave/brave-core/pull/20634/commits/14fd36bf729975cdbf628608b1de08abdeee5272